testing/e2e/app/routes/Table/route.tsx

```diff
@@ -121,14 +141,21 @@ export default function Resizing() {
             width: '8rem',
           },
         ]}
-        data={data}
+        data={enableVirtualization ? virtualizedData : data}
         emptyTableContent='No data.'
         isResizable
         isRowDisabled={isRowDisabled}
         isSelectable={isSelectable}
         isSortable
         columnResizeMode={columnResizeMode as 'fit' | 'expand' | undefined}
         selectSubRows={selectSubRows}
+        enableVirtualization={enableVirtualization}
+        style={enableVirtualization ? { maxHeight: '90vh' } : undefined}
+        scrollToRow={
+          scroll
+            ? (rows, data) => rows.findIndex((row) => row.original === data[50])
```

Comment:
nit: It would be better if the number `50` was passed using the query params, so that the test case reads more naturally.

Reply:
i meant replacing the boolean param with a number param, but i guess we can have both